docs/reference/run.md

```diff
@@ -624,6 +624,8 @@ container:
 | `--cpu-quota=0`            | Limit the CPU CFS (Completely Fair Scheduler) quota                                         |
 | `--blkio-weight=0`         | Block IO weight (relative weight) accepts a weight value between 10 and 1000.               |
 | `--blkio-weight-device=""` | Block IO weight (relative device weight, format: `DEVICE_NAME:WEIGHT`)                                                |
+| `--read-bps-device="" `    | Specifies upper limit on READ rate from the device.                                         |
+| --write-bps-device="" `    | Specifies upper limit on WRITE rate to the device.                                          |
```

Comment:
Missing tic here.  Also, is it necessary to actual specify that it is an _upper_ --- are their conditions under which I would specify the slowest a read or write rate allowed?  If no, "limit" itself should suffice:  

```
| `--read-bps-device="<device-path>:<limit>[unit]" `    | Limit the read rate from a device.  Limit is a positive integer. Unit can be one of kb, mb, or gb.
| `--write-bps-device="<device-path>:<limit>[unit]" `    | Limit the write rate to a device.  Limit is a positive integer. Unit can be one of kb, mb, or gb.
```